Dockerfile

```diff
@@ -10,4 +12,5 @@ EXPOSE 4443
 
 WORKDIR /go/src/github.com/docker/notary
 
-CMD [ "notary-server", "-config", "cmd/notary-server/dev-config.json" ]
+
+CMD [ "notary-server", "-config", "cmd/notary-server/stage-config.json" ]
```

Comment:
(unrelated) Just noticed that notary uses a single dash for non-shorthand options, i.e. `-options`, not `--options`? This is different from Docker itself, which uses `--` for non-shorthand options.